src/types/parameter/parameter_enums.enums.yaml

```diff
@@ -28,6 +31,9 @@ enums:
       - name: Length_Error
         value: 3
         description: Parameter length was not correct.
+      - name: Validation_Success
+        value: 4
+        description: Parameter values were successfully validated.
```

Comment:
Typically we only include a single enumeration value for "success", and everything else can be assumed to be an off nominal return. Is there an issue with using the Success/0 in this case to indicate successful validation?

Reply:
No issue, it makes sense to remove this. Addressed in [9328969](https://github.com/lasp/adamant/pull/64/commits/9328969af2f37b9ecf0b0c96d383d93f2ecd69b8).